src/frontend/src/views/Data/components/TreeActionMenu/components/MoreActionsMenu/index.tsx

```diff
@@ -2,19 +2,40 @@ import { Menu, MenuItem, Tooltip } from "czifui";
 import React, { MouseEventHandler, useState } from "react";
 import { TREE_STATUS } from "src/common/constants/types";
 import MoreActionsIcon from "src/common/icons/IconDotsHorizontal3Large.svg";
-import { StyledIcon } from "../../style";
+import { UserResponse } from "src/common/queries/auth";
+import { StyledIcon, StyledIconWrapper } from "../../style";
 import { StyledText, StyledTrashIcon } from "./style";
 
 interface Props {
-  item: TableItem;
+  item: Tree;
+  onDeleteTreeModalOpen(t: Tree): void;
+  userInfo: UserResponse;
 }
 
-const MoreActionsMenu = ({ item }: Props): JSX.Element => {
+const MoreActionsMenu = ({
+  item,
+  onDeleteTreeModalOpen,
+  userInfo,
+}: Props): JSX.Element => {
   const [anchorEl, setAnchorEl] = useState<Element | null>(null);
 
-  const { status } = item;
-  //TODO: also disable if cdph viewing
-  const isDisabled = status === TREE_STATUS.Started;
+  const { group: userGroup } = userInfo;
+  const { group, status } = item;
+
+  const isAutoBuild = group?.name === "";
+  const isTreeInUserOrg = userGroup?.name === group?.name;
+  const canUserDeleteTree = isAutoBuild || isTreeInUserOrg;
```

Comment:
just checking, are users allowed to delete auto builds? I thought we landed against this but i could be wrong.

Reply:
I thought we would allow them to delete the trees if they wanted, but not in bulk. Let me double check.

Reply:
@Janeece or @jgadling do you remember?

Reply:
i couldn't find any details in the tickets that included this as an AC

Reply:
I don't see any reason they shouldn't be able to delete auto builds, but I wasn't part of those conversations.

Reply:
https://czi-sci.slack.com/archives/C01G8BDSUM9/p1640029573174300

Reply:
We're good to go deleting all tree types